unit2.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Ghibli Review App</title>
+    <link rel="stylesheet" href="./unit2.css">
+</head>
+
+<body>
+    <div id="header">
+        <img id="logo"
+            src="https://upload.wikimedia.org/wikipedia/en/thumb/c/ca/Studio_Ghibli_logo.svg/1200px-Studio_Ghibli_logo.svg.png"
+            alt="Studio Ghibli Logo">;
+    </div>
+
+    <div id = "containerForSelectBox">
+    <form>
+        <select id="mySelect">
+        </select>
+        <br><br>
+        <input type="button" onclick="getOption()" value="Click Me!">
```

Comment:
Instead of having separate button to select a movie and get a data, we could put "onchange = "getOption()"" on select tag. onchange recognizes when a use chooses one of options automatically.